client/app/billing/autoRenew/actions/update/manualPayment/translations/Messages_fr_FR.json

```diff
@@ -1,7 +1,7 @@
 {
   "billing_autorenew_service_update_manual_sumup": "Vous avez sélectionné le renouvellement manuel pour votre service {{ serviceName }}",
   "billing_autorenew_service_update_manual_non_automatic": "Dans ce mode de renouvellement, votre service n'est pas automatiquement reconduit lorsqu'il arrive à échéance.",
-  "billing_autorenew_service_update_manual_expiration": "Plusieurs notifications vous seront envoyées par mails durant les semaines/jours précédents son terme, afin de vous prévenir de son expiration le {{ expiration }}.",
+  "billing_autorenew_service_update_manual_expiration": "Plusieurs notifications vous seront envoyées par mails durant les semaines/jours précédant son terme, afin de vous prévenir de son expiration le {{ expiration }}.",
```

Comment:
Is this text has been validated by content?

I personally prefer rename `mails` into `e-mail`.

Do you agree?

Reply:
This is the text that was provided by CX in the mock ups

Reply:
`CX` !== `Content`, `Content` wants us to write it `e-mail` ;)